### DISCUSSION TITLE
Add Latest or Master tag?

### DISCUSSION THREAD

**Discussion by wuast94** [wazuh/wazuh-docker#1190] [General]
Why is there no tag to keep the Container updated? version pinning is ok to dont break anything.
But its also a nightmare to keep track of updates, if its just one project ok, but if you have multiple containers that dont have latest stable master or anything tag its just a nightmare to monitor anything for updates, update the containers/compose etc

**Comment by rauldpm** [comment]
Hello @wuast94 

Wazuh follows the rule that the master branch is the `latest development branch`, which is specified in the README.md file

![image](https://github.com/wazuh/wazuh-docker/assets/14913942/0abadf9c-6acf-4ced-8bfb-ac80dc4b704e)

> [!NOTE]
> Do not mind the stable branch, as it was deprecated and the README must be updated.

If you want the latest changes, please refer to the tags that are only composed by `vMAJOR.MINOR.PATCH` (`vX.Y.Z`) and if you want to know which one is the latest, please refer to the [Documentation release notes](https://documentation.wazuh.com/current/release-notes/index.html), if you want to check it in the repository, please refer to the `Releases` section

![image](https://github.com/wazuh/wazuh-docker/assets/14913942/8cf54cc3-9f53-45ff-9097-d4d44ddc03a5)

**Reply by wuast94** [reply]
ok so i have to monitor the releases to get updates on updates and then manually go to my compose and update it there 😊
i am more a fan of a dynamic tag that gets updated and dont run on a old base etc, running "old" containers is always a security concern.
but know i know it and have to monitor for new releases 😊

**Reply by rauldpm** [reply]
As I said in https://github.com/wazuh/wazuh-docker/issues/1191, yes, you can check changes using the branches/tags but if you are going to deploy it, please follow the Wazuh documentation https://documentation.wazuh.com/current/deployment-options/docker/index.html
